### PR TITLE
Ticket 25/get put delete route user

### PR DESCRIPTION
**Pull request by seintun**

### CODE REVIEW

File: controllers/user.ctrl.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// UpdateUserByID by ID
+func UpdateUserByID(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
+	var u User
+	if err := json.NewDecoder(r.Body).Decode(&u); err != nil {
+		respondWithError(w, http.StatusBadRequest, "Invalid request payload")
+		return
+	}
+	params := mux.Vars(r)
+	err := dao.UpdateUserByID(params["id"], u)
```
Comment (developerX) [inline controllers/user.ctrl.go]: Something wrong here!

**Comment by developerX** [comment]
Got this to work1

**Comment by developerX** [comment]
Got this to work but the PUT request needs to have the id passed into the body of the request rather then using the url params.

**Comment by seintun** [comment]
Created a different ticket for UPDATE @ ticket#32

**Pull request merged**